setup.py

```diff
@@ -33,7 +33,6 @@
 Programming Language :: Python :: 2
 Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
-Programming Language :: Python :: 3.3
 Programming Language :: Python :: 3.4
 Programming Language :: Python :: 3.5
 """
```

Comment:
I'm not sure we support it well. As I think the master have travis failing, I prefer not to do that in this PR.

There was many report of problems of Python 3.6, so until I got someone report that it work on the 3 platforms, I prefer we don't do it. It could be just 1 days of testing...